pkgs/development/python-modules/sphinx-sitemap/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, pytest
+,
+}:
+let
+  pname = "sphinx-sitemap";
+  version = "2.5.1";
+in
+buildPythonPackage {
+  inherit pname version;
```

```suggestion
  inherit pname version;
  pyproject = true;
```

Use the PEP517 builder. This might requires `setuptools` in `nativeBuildInputs`.